taskcluster/ci/test/kind.yml

```diff
@@ -70,3 +70,24 @@ jobs:
                 - ['automation/taskcluster/androidTest/ui-test.sh', 'samples-glean', 'arm', '1']
         treeherder:
             symbol: 'ui-samples-glean'
+    android-feature-pwa:
```

Comment:
@pocmo you can see all the new test running on this branch https://github.com/mozilla-mobile/android-components/commits/android_test on this commit https://github.com/mozilla-mobile/android-components/commit/945ffee42b6a77e6ab98229f9884157f2df7fd79